app/models/lookup_key.rb

```diff
@@ -158,6 +158,10 @@ def path_elements
     end
   end
 
+  def contains_erb?(value)
```

Comment:
Should this also depend on the value of the interpolate_erb_in_parameters setting?